src/views/webview-app/use-connection-form.ts

```diff
@@ -145,7 +145,9 @@ export default function useConnectionForm() {
       }
     };
     window.addEventListener('message', handleConnectResultResponse);
-    () => window.removeEventListener('message', handleConnectResultResponse);
+    return () => {
+      window.removeEventListener('message', handleConnectResultResponse);
```

Comment:
Was this not working before? 🙈 
Nice catch

Reply:
It was printing warnings in the terminal that I guess we just ignored 🙈